hack/netpol/pkg/utils/k8s_util.go

```diff
@@ -192,6 +192,22 @@ func (k *Kubernetes) CreateOrUpdateNamespace(n string, labels map[string]string)
 func (k *Kubernetes) CreateOrUpdateDeployment(ns, deploymentName string, replicas int32, labels map[string]string) (*appsv1.Deployment, error) {
 	zero := int64(0)
 	log.Infof("creating/updating deployment %s in ns %s", deploymentName, ns)
+	makeContainerSpec := func(port int32) v1.Container {
+		return v1.Container{
+			Name:            fmt.Sprintf("c%d", port),
+			Image:           "busybox:latest",
+			Command:         []string{"/bin/sh", "-c"},
+			Args:            []string{fmt.Sprintf("touch index.html && httpd -f -p %d", port)},
+			SecurityContext: &v1.SecurityContext{},
+			Ports: []v1.ContainerPort{
+				v1.ContainerPort{
+					ContainerPort: port,
+					Name:          fmt.Sprintf("serve-%d", port),
```

Comment:
not related to your patch but i didn't quite like this in upstream either where the name of the port was related to port number.. can't really test the named port support effectively with this.. we can update when we add named port test cases

Reply:
I'm going to let you guys figure it out :)

Reply:
+1 abishek to get a better story around named ports